src/app/restaurant/restaurant.component.html

```diff
@@ -4,4 +4,46 @@
   <p>Description: {{ restaurant.description }}</p>
 
   <p>Date of Creation: {{ restaurant.dateOfCreation }}</p>
+  <form [formGroup]="reviewForm">
```

Comment:
Form submission can be done on the form element itself by using the (onSubmit) handler. This will allow user to press enter or submit the form other than on click as it is now implemented. With this we can remove the click handler from the button and just add a `type="submit"` for the button. Submit is the default type of the button but it is good to note it specifically for better DX